package.json

```diff
@@ -27,11 +27,10 @@
   "dependencies": {
     "@fastify/multipart": "6.0.0",
     "@fastify/static": "5.0.2",
+    "@graasp/plugin-file": "github:graasp/graasp-plugin-file",
```

Comment:
So renamed it just here, not in the package? Should we create an issue in the repo to rename it?

Reply:
Ah i see... maybe it's a good idea. It seems fastify followed this pattern (https://github.com/fastify/fastify-websocket/blob/master/package.json) so their repo names are `fastify/fastify-*` and their package names look like `@fastify/*`.
Anyway, I think we can start by normalizing the dependencies aliases, so that in each repos imports and upgrades always use the same format.